tools/sof-fw-boot-test.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+
+# define command line arguments
+def parse_cmdline():
+	parser = argparse.ArgumentParser(add_help=True, formatter_class=argparse.RawTextHelpFormatter,
+	    description='A script for stress testing firmware boot')
+
+	parser.add_argument('-i', '--iter', type=int, default=100, help="number of firmware boot iterations")
+	parser.add_argument('-f', '--firmware', type=str, required=True, help='firmware filename')
+	parser.add_argument('-p', '--fw_path', type=str, required=True, help='path to the firmware file relative to /lib/firmware')
+
+	return vars(parser.parse_args())
+
+cmd_args = parse_cmdline()
+
+# Get firmware file path
+fw_path = cmd_args["fw_path"]
```

Comment:
How about using the new `/sys/kernel/debug/sof/fw_profile/` instead?

Reply:
fw_profile is readonly meant for the user to see the firmware info on a working device. I dont think we should mix the testing profile with that of the real profile